src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp

```diff
@@ -682,7 +671,7 @@ void CompositionalMultiphaseWell::updateBHPForConstraint( WellElementSubRegion &
 
 }
 
-void CompositionalMultiphaseWell::updateVolRatesForConstraint( WellElementSubRegion & subRegion )
+void CompositionalMultiphaseWell::updateVolRatesForConstraint( ElementRegionManager const & elemManager, WellElementSubRegion & subRegion )
```

Comment:
(same for `SinglePhaseWell`)
```suggestion
void CompositionalMultiphaseWell::updateVolRatesForConstraint( ElementRegionManager const & elemManager,
                                                               WellElementSubRegion & subRegion )
```